tests/meson.build

```diff
@@ -0,0 +1,15 @@
+test_inc = include_directories('.')
+
+prog_src = [
+    'errors.cpp',
+    'file_helper.cpp',
+    'file_log.cpp',
+    'format.cpp',
+    'registry.cpp',
+    'test_macros.cpp',
+    'utils.cpp',
+    'main.cpp'
+]
+
+tests_exe = executable('tests_exe', prog_src, include_directories : [inc, test_inc], dependencies : thread_dep)
```

Comment:
You don't need to to add `.` to the search path. Meson guarantees that the current source and binary directories are always in header search path. This was also incorrect in the 11 branch, but let's fix it now. Thanks.